client/components/Mentions.vue

```diff
@@ -8,10 +8,10 @@
 		<div class="mentions-popup">
 			<div class="mentions-popup-title">
 				Recent mentions
-				<span v-if="isLoading">- Loading…</span>
 			</div>
 			<template v-if="resolvedMessages.length === 0">
-				<p>There are no recent mentions.</p>
+				<span v-if="isLoading">Loading…</span>
+				<p vu-if="!isLoading">There are no recent mentions.</p>
```

Comment:
Typo.
```suggestion
				<p v-else>There are no recent mentions.</p>
```

Reply:
I'm surprised this isn't caught by tests & lint :thinking:

Reply:
Well it's just an attribute, how will the linter know?

Reply:
I thought it might check all valid HTML5 and Vue attributes as well as `data-*` but I guess not.

Reply:
Fixed

Reply:
Can you squash please